src/iop/rawprepare.c

```diff
@@ -574,20 +606,44 @@ static int image_is_normalized(const dt_image_t *const image)
   return image->buf_dsc.channels == 1 && image->buf_dsc.datatype == TYPE_FLOAT;
 }
 
-static gboolean image_set_rawcrops(const uint32_t imgid, int dx, int dy)
+static gboolean _image_set_rawcrops(
+        dt_iop_module_t *self,
+        const uint32_t imgid,
+        int left,
+        int right,
+        int top,
+        int bottom)
 {
   dt_image_t *img = NULL;
   img = dt_image_cache_get(darktable.image_cache, imgid, 'r');
-  const gboolean test = (img->p_width == img->width - dx)
-                     && (img->p_height == img->height - dy);
+
+  const gboolean cropvalid = (left >= 1) && (right >= 0) && (top >= 0) && (bottom >= 0)
```

Comment:
Just spotted this now, `left >= 0` , no?

Reply:
Absolutely. Had this while testing for the trouble message and forgot.

I am on the bike, feel free to fix.

Btw, one more idea while riding, we have the libraw open now, some people were overriding the cr3 by renaming. All libraw files might introduce bugs...

Reply:
> I am on the bike, feel free to fix.

Fixed! So you are also a Cyclist :)